lib/sorcery/model/temporary_token.rb

```diff
@@ -7,12 +7,14 @@ module Model
     # such as reseting password and activating the user by email.
     module TemporaryToken
       def self.included(base)
+        # FIXME: This may not be the ideal way of passing sorcery_config to generate_random_token.
+        @sorcery_config = base.sorcery_config
```

Comment:
`@sorcery_config` is owned by `TemporaryToken`.
It will be overridden if we include `TemporaryToken`.

```ruby
module Foo
  def self.included(base)
    @var = base.name
  end

  def self.foo
    puts @var
  end
end

class Bar
  include Foo
end

Foo.foo #=> Bar
puts Foo.instance_variable_get(:@var) #=> Bar

class Baz
  include Foo
end

Foo.foo #=> Baz
puts Foo.instance_variable_get(:@var) #=> Baz
```

Is this intentional behaviour?